app/boards/intel_adsp_ace20_lnl.conf

```diff
@@ -0,0 +1,71 @@
+CONFIG_LUNARLAKE=y
+CONFIG_IPC_MAJOR_4=y
+
+# turn off SOF drivers
+CONFIG_INTEL_DMIC=n
+CONFIG_INTEL_DMIC_NHLT=n
+CONFIG_INTEL_SSP=n
+CONFIG_INTEL_ALH=n
+CONFIG_COMP_SRC=y
+
+CONFIG_COMP_SRC_IPC4_FULL_MATRIX=y
+
+# power settings
+CONFIG_PM=y
+CONFIG_PM_DEVICE=y
+CONFIG_PM_DEVICE_RUNTIME=y
+CONFIG_PM_DEVICE_POWER_DOMAIN=y
+CONFIG_PM_POLICY_CUSTOM=y
+
+CONFIG_POWER_DOMAIN=y
+CONFIG_POWER_DOMAIN_INTEL_ADSP=y
+
+# enable Zephyr drivers
+CONFIG_ZEPHYR_NATIVE_DRIVERS=y
+CONFIG_DAI=y
+CONFIG_DAI_INIT_PRIORITY=70
+CONFIG_DAI_INTEL_DMIC=y
+CONFIG_DAI_DMIC_HW_IOCLK=19200000
```

Comment:
this is only true on development systems.

Reply:
Added fpga_overlay.